tests/foreman/rhai/test_rhai.py

```diff
@@ -68,16 +68,20 @@ def test_positive_register_client_to_rhai(self):
         """
         # Register a VM to Access Insights Service
         with VirtualMachine(distro='rhel67') as vm:
-            vm.configure_rhai_client(self.ak_name, self.org_label, 'rhel67')
-
-            with Session(self.browser) as session:
-                # view clients registered to Red Hat Access Insights
-                session.nav.go_to_select_org(self.org_name)
-                Navigator(self.browser).go_to_insights_systems()
-                result = self.rhai.view_registered_systems()
-                self.assertIn("1", result, 'Registered clients are not listed')
-            vm.get('/var/log/redhat-access-insights/redhat-access'
-                   '-insights.log', './insights_client_registration.log')
+            try:
+                vm.configure_rhai_client(self.ak_name, self.org_label,
+                                         'rhel67')
+
+                with Session(self.browser) as session:
+                    # view clients registered to Red Hat Access Insights
+                    session.nav.go_to_select_org(self.org_name)
+                    Navigator(self.browser).go_to_insights_systems()
+                    result = self.rhai.view_registered_systems()
+                    self.assertIn("1", result,
+                                  'Registered clients are not listed')
+            finally:
+                vm.get('/var/log/redhat-access-insights/redhat-access'
```

Comment:
@elyezer I changed the formatting to above because it was giving >80 too long error on flak8 check. So, if I change it to what you mentioned, it will not pass the flak8 checks.